app/views/adopters/steps/_your_home.html.erb

```diff
@@ -1,62 +1,92 @@
+        <div class="controls">
+            <label>
+                <%= af.check_box :other_household_names, {}, "Yes", "No" %>
+            </label>
+            <div id="household_names" style="display:none;">
+                <%= af.text_field :other_household_names %>
+                <span>Please Enter Name and Age</span>
+            </div>
+            <div id ="add_person">
+                <a>Add Person</a>
+            </div>
+        </div>
+        <script type ="text/javascript">
+            var checkbox = document.getElementById('adopter_adoption_app_attributes_other_household_names');
+            var details_div = document.getElementById('household_names');
+            checkbox.onchange = function() {
+                if(this.checked) {
+                    details_div.style['display'] = 'block';
+                }  else {
+                    details_div.style['display'] = 'none';
+                }
+            };
+            
+            
+            document.getElementById('add_person').onclick = function(){
+                details_div.innerHTML = details_div.innerHTML + '<input type="text">';
+            };
+
+        </script>
+
+
     </div>
-  </div>
-  <div class="control-group">
-    <%= af.label :house_type, "Do you rent or own your home?", :class => "control-label" %>
-    <div class="controls">
-      <label>
-        <%= af.radio_button :house_type, 'rent', :class => "link required residence-auth-pop" %>
-        <span>Rent</span>
-      </label>
-      <label>
-        <%= af.radio_button :house_type, 'own', :class => "link required residence-auth-pop" %>
-        <span>Own</span>
-      </label>
+
+    <div class="control-group">
+        <%= af.label :house_type, "Do you rent or own your home?", :class => "control-label" %>
+        <div class="controls">
+            <label>
+                <%= af.radio_button :house_type, 'rent', :class => "link required residence-auth-pop" %>
+                <span>Rent</span>
+            </label>
+            <label>
+                <%= af.radio_button :house_type, 'own', :class => "link required residence-auth-pop" %>
+                <span>Own</span>
+            </label>
+        </div>
     </div>
-  </div>
-  <div class="control-group own-home-auth">
-    <%= af.label :verify_home_auth, "I authorize Operation Paws for Homes to verify my home ownership.", :class => "control-label" %>
-    <div class="controls">
-      <label class="checkbox">
-        <%= af.check_box :verify_home_auth, :class => "required"  %>
-      </label>
+    <div class="control-group own-home-auth">
+        <%= af.label :verify_home_auth, "I authorize Operation Paws for Homes to verify my home ownership.", :class => "control-label" %>
+        <div class="controls">
+            <label class="checkbox">
+                <%= af.check_box :verify_home_auth, :class => "required"  %>
+            </label>
+        </div>
     </div>
-  </div>
 </div>
```

Comment:
What editor/IDE are you using and how to do have it configured?

Reply:
I am using brackets on my mac. I don't think I have anything special configured with it.

Reply:
Okay, there should be a setting in Brackets to modify the indentations from "Spaces: 4" to "Spaces: 2"  which is what this project uses.    It looks like when you editing this file it changed the formatting from the existing 2 Space indentation to a 4 Space Indentation.  

https://www.quora.com/How-does-one-change-the-indent-space-in-the-Brackets-code-editor